areas.wotw

```diff
@@ -850,6 +873,8 @@ anchor HowlsDen.Teleporter at -328, -4536:  # At the teleporter
       SentryBreak=16, DoubleJump OR Launch OR Bash OR Sword OR Hammer
       HammerBreak, DoubleJump OR Launch OR Bash OR Sword OR Hammer
       SpearBreak, DoubleJump OR Launch OR Bash OR Sword OR Hammer
+  state HowlsDen.DoubleJumpTreeCombat:
+    moki: Combat=Mantis+2xLizard+SmallSkeeto  # the game puts this before regen. It's why all the enemies drop tons of health. There's more enemy here once you get Bash.
```

Comment:
This changer also impact assumptions made in other places.
E.g. the conncetion to HowlsDen.DoubleJumpTreeArea does not have a Sword/Hammer path because that would have been redundant with the fight arena.
The addition of an aerial enemy obsoletes this assumption, requiring changes to the surrounding connections as well